### DISCUSSION TITLE
ipxe.efi : could not install base code protocol: Error 0x7f4ae082

### DISCUSSION THREAD

**Discussion by karawitan** [ipxe/ipxe#342] [General]
Hi !

   while trying to boot vmlinuz kernel from Fedora Silverblue 33, over PXE using iPXE;
I get up to the step where vmlinuz is downloaded, but then I get the "could not install base code protocol: Error 0x7f4ae082" message, as shown in the picture below:

![PXE](https://user-images.githubusercontent.com/37262984/116286228-42ee2f80-a78f-11eb-9d1e-6c554682ac78.jpg)

( This is a debug build ) 

any hint ?

**Comment by NiKiZe** [comment]
"Could not boot image: Error 0x7f4ae082 (http://ipxe.org/7f4ae082)"
Is that correct? Did you visit that URL? Or any particular reason you left the URL out of the initial post?

**Reply by NiKiZe** [reply]
So, pure your EFI firmware returns the error, is secureboot disabled? Have you tried to chain to and from any other EFI binary? For example efishell or wimboot or other ipxe.efi?

**Reply by karawitan** [reply]
> "Could not boot image: Error 0x7f4ae082 (http://ipxe.org/7f4ae082)"
> Is that correct? Did you visit that URL? Or any particular reason you left the URL out of the initial post?

Yep sure, I visited this url, that's why I prepared a debug build. Maybe should I increase the debug build verbosity ?

**Reply by karawitan** [reply]
> So, pure your EFI firmware returns the error, is secureboot disabled? Have you tried to chain
> to and from any other EFI binary? For example efishell or wimboot or other ipxe.efi?

The hardware I'm trying to boot is an iMac 24 early 2008 aka iMac8,1. It does not have a T2 chip, hence I think it does not implement secureboot.

**Reply by karawitan** [reply]
ok as far as I understand, and after reading "[A Brief History of Apple and EFI](http://refit.sourceforge.net/info/apple_efi.html)", Seems that this iMac has EFI 1.10 (as stated in the “Santa Rosa” Models chapter)
